### ISSUE TITLE
Lens: Drawing focal points always fails

### ISSUE THREAD

**Issue by genodeftest** [fruchart/tikz-optics#2]
Steps to reproduce:
1. use the latest version of texlive
2. in a tikz environment, use a lens
3. on the lens, set `draw focal points` to any value (does not matter)

What happens:
I get this error message:

```
! Package pgfkeys Error: I do not know the key '/tikz/cross out' and I am going
 to ignore it. Perhaps you misspelled it.
```

What should happen:
Compile without issues.

Additional info:
According to the pgfmanual.pdf (`texdoc tikz`), `/tikz/cross out` does not exist as it is not mentioned anywhere.

Version info:
pdflatex.exe --version
pdfTeX 3.14159265-2.6-1.40.18 (TeX Live 2017/W32TeX)
kpathsea version 6.2.3
Copyright 2017 Han The Thanh (pdfTeX) et al.
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
Compiled with libpng 1.6.29; using libpng 1.6.29
Compiled with zlib 1.2.11; using zlib 1.2.11
Compiled with xpdf version 3.04

tikz-optics: 43466 (0.2.3)
pgf: 44231 (3.0.1a)

**Comment by genodeftest** [comment]
Workaround:
In [tikzlibraryoptics.code.tex#L2112](https://github.com/fruchart/tikz-optics/blob/master/tikzlibraryoptics.code.tex#L2112), replace `cross out` by something else, e.g. `circle`.

**Comment by fruchart** [comment]
Thanks for the report! The tikz library `shapes.misc` was not called, this is corrected by b27800b. You can use `\usetikzlibrary{shapes.misc}` to make it work without updating the library.

(By the way, the shape `cross out` is indeed explained on page 734 of the TikZ manual, version 3.0.1a.)

**Comment by genodeftest** [comment]
Thanks for this quick fix! & sorry for the noise. Looks like I failed to use Ctrl+F correctly.
